Split/Models/Matcher.swift

```diff
@@ -21,7 +21,23 @@
     var dependencyMatcherData: DependencyMatcherData?
     var booleanMatcherData: Bool?
     var stringMatcherData: String?
-    weak var client: InternalSplitClient?
+    private let clientQueue = DispatchQueue(label: "split-condition", target: DispatchQueue.global())
```

Comment:
is it okey using the same `split-condition` label than [here](https://github.com/splitio/ios-client/pull/221/files?file-filters%5B%5D=.swift&file-filters%5B%5D=.xcscheme#diff-d0afe4628a040368441c664f6d4b281598283e72d877c0f297e58f2de10afb9cR17)?